q.yaml

```
@@ -0,0 +1,36 @@
+package:
+  name: q
+  version: 0.11.4
+  epoch: 0
+  description: "A tiny command line DNS client with support for UDP, TCP, DoT, DoH, DoQ and ODoH."
+  copyright:
+    - license: GPL-3.0
```

It appears the `GPL-3.0` identifier is deprecated, as per https://spdx.org/licenses/

```suggestion
    - license: GPL-3.0-only
```